### PR TITLE
Renamed 'overview' to 'browse'

### PR DESCRIPTION
**Pull request by julen**
As discussed on IRC, PRing for a good to go. Also replaces internal references besides user-facing strings.

### CODE REVIEW

File: pootle/apps/pootle_language/views.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
                                  get_table_headings)
 from pootle.core.decorators import get_path_obj, permission_required
 from pootle.core.helpers import (get_export_view_context,
-                                 get_overview_context,
+                                 get_browser_context,
```
Comment (unho) [inline pootle/apps/pootle_language/views.py]: `browser`?

File: pootle/apps/import_export/templates/browser/index.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{% overextends "browser/overview.html" %}
+{% overextends "browser/index.html" %}
```
Comment (unho) [inline pootle/apps/import_export/templates/browser/index.html]: If you are renaming perhaps it should be `browse/index.html`.

File: pootle/templates/editor/_actions.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <ul>
   <li class="back-to-browser">
     <a class="js-back-to-browser" href="{{ previous_url }}">
-      <span class="caption">{% trans 'Go back to overview' %}</span>
+      <span class="caption">{% trans 'Go back to browser' %}</span>
```
Comment (unho) [inline pootle/templates/editor/_actions.html]: Didn't someone raised the concern that browser can be misleading?

File: pootle/apps/pootle_store/urls.py [hunk 4]
```diff
@@ -19,9 +19,9 @@
     url(r'^xhr/stats/checks/?$',
         'get_qualitycheck_stats',
         name='pootle-xhr-stats-checks'),
-    url(r'^xhr/stats/overview/?$',
-        'get_overview_stats',
-        name='pootle-xhr-stats-overview'),
+    url(r'^xhr/stats/?$',
```
Comment (unho) [inline pootle/apps/pootle_store/urls.py]: Just wondering why we keep the `?`. These are XHR calls so we can enforce these URLs to always end on `/` on our JS code.

**Comment by unho** [comment]
Finished review. Just noticed some bits that I am not 100% ok with, but lgtm anyway.

**Comment by dwaynebailey** [comment]
Similar concerns to @unho when we use browser instead of browse.  As it's a potential confusion between the webbrowser and our project browser.

Otherwise gtm

File: pootle/apps/pootle_language/views.py (hunk 1 above)
Comment (julen) [inline pootle/apps/pootle_language/views.py]: yes, in line with `get_translation_context`.

File: pootle/apps/import_export/templates/browser/index.html (hunk 2 above)
Comment (julen) [inline pootle/apps/import_export/templates/browser/index.html]: This is in line with *templates/editor*.

File: pootle/templates/editor/_actions.html (hunk 3 above)
Comment (julen) [inline pootle/templates/editor/_actions.html]: Yes, and we are fine with that.

File: pootle/apps/pootle_store/urls.py (hunk 4 above)
Comment (julen) [inline pootle/apps/pootle_store/urls.py]: Irrelevant here.

File: pootle/templates/editor/_actions.html (hunk 3 above)
Comment (julen) [inline pootle/templates/editor/_actions.html]: I mean, what is the file/dir browser? Can you name it otherwise? cc @iafan.

Comment (julen) [inline pootle/templates/editor/_actions.html]: I can think of *Go back to browsing* or *Go back to browse view* (also, *mode* instead of *view*).

Comment (unho) [inline pootle/templates/editor/_actions.html]: Any of those two (*Go back to browsing* or *Go back to browse view*) is fine for me.

Comment (iafan) [inline pootle/templates/editor/_actions.html]: +1 for _Go back to browsing_

File: pootle/apps/import_export/templates/browser/index.html (hunk 2 above)
Comment (jleclanche) [inline pootle/apps/import_export/templates/browser/index.html]: I agree the "browser" wording is a bit confusing.

**Comment by jleclanche** [comment]
gtm but i share @unho and @dwaynebailey's concerns.

**Comment by jleclanche** [comment]
Woops I'm too late.

**Pull request merged**